javaagent-exporters/logging/src/main/java/io/opentelemetry/javaagent/exporters/logging/LoggingExporter.java

```diff
@@ -21,28 +25,37 @@ public LoggingExporter(String prefix) {
 
   @Override
   public CompletableResultCode export(Collection<SpanData> list) {
+    StringBuilder stringBuilder = new StringBuilder();
     for (SpanData span : list) {
-      System.out.print(
-          prefix + " " + span.getName() + " " + span.getTraceId() + " " + span.getSpanId() + " ");
+
+      stringBuilder
+          .append(prefix)
+          .append(" ")
+          .append(span.getName())
+          .append(" ")
+          .append(span.getTraceId())
+          .append(" ")
+          .append(span.getSpanId())
+          .append(" ");
+
       span.getAttributes()
           .forEach(
               new AttributeConsumer() {
                 @Override
                 public <T> void consume(AttributeKey<T> key, T value) {
-                  System.out.print(key + "=");
-                  switch (key.getType()) {
-                    case STRING:
-                      System.out.print('"' + String.valueOf(value) + '"');
-                      break;
-                    default:
-                      System.out.print(value);
-                      break;
+
+                  stringBuilder.append(key).append('=');
```

Comment:
```suggestion
                  stringBuilder.append(key.getKey()).append('=');
```

Thanks for bringing up the issue about toString, I think getKey should solve that